bindings/matrix-sdk-ffi/src/client.rs

```diff
@@ -573,11 +571,14 @@ impl Client {
     /// Be careful that only the most recent value can be observed. Subscribers
     /// are notified when a new value is sent, but there is no guarantee that
     /// they will see all values.
+    ///
+    /// Doesn't work for `m.secret_storage.key.<key_id>`, because it is
+    /// currently not observable.
```

Comment:
Note that we are adding stuff to Ruma that should make this possible in the future (https://github.com/ruma/ruma/pull/2136), although the event handler code will need to be adapted in the SDK.

So maybe it's better not to touch the function signature and panic for now?